lib/opentracing/scope_manager.rb

```diff
@@ -0,0 +1,35 @@
+module OpenTracing
+  # ScopeManager represents an OpenTracing ScopeManager
+  #
+  # See http://www.opentracing.io for more information.
+
+  # The ScopeManager interface abstracts both the activation of Span instances
+  # via ScopeManager#activate and access to an active Span/Scope via
+  # ScopeManager#active
+
```

Comment:
Probably need to remove this line or add `#`. I think otherwise there might be some weirdness generating docs.